src/main/kotlin/org/jitsi/nlj/transform/node/ToggleablePcapWriter.kt

```diff
@@ -29,15 +29,27 @@ class ToggleablePcapWriter(
 ) {
     private var pcapWriter: PcapWriter? = null
 
+    @Synchronized fun enable() {
+        if (pcapWriter == null) {
+            pcapWriter = PcapWriter(parentLogger, "/tmp/$prefix-${Date().toInstant()}.pcap")
+        }
+    }
+
+    @Synchronized fun disable() {
+        if (pcapWriter != null) {
+            pcapWriter?.close()
```

Comment:
This shouldn't need `?.` inside a `!= null` clause, right?

Reply:
yeah, that's what i thought as well but kotlin still complains. Maybe because i'm accessing a field and the compiler is not smart enough to see the synchronized?

Reply:
Ah, that makes sense.

Reply:
This check is unnecessary, you can just do `pcapWriter?.close`

Reply:
done